classroom.rb

```diff
@@ -14,24 +14,36 @@ def add_student
 end
 
 class Book
+  @@books = []
   def initialize(title, author)
     @title = title
     @author = author
+    @@books.push({ 'title' => @title, 'author' => @author })
   end
   attr_accessor :title, :author
 
   def add_rental(date)
     new.Rental(date)
   end
+
+  def list_of_books
+    @@books.each_with_index do |book, idx|
+      puts "#{idx}) Title: #{book['title']}, Author: #{book['author']}"
+    end
+  end
+
+  def books_array
+    @@books
+  end
```

Comment:
- [ ] Can you elaborate why we need this method? And if we need it it has to go the App class as well.